code-of-conduct.md

```diff
@@ -1,38 +1,134 @@
+- Unwelcome comments regarding a person’s lifestyle choices and practices,
+including those related to food, health, parenting, relationships, drugs, and
+employment
+- Unwelcome sexual attention or advances
+- Other conduct that is inappropriate for a professional audience that includes
+people of many different backgrounds
+
+Participants asked to stop any inappropriate behavior are expected to comply
+immediately.
+
+If a participant engages in behavior that violates this code of conduct, the
+event organizers may take any action they deem appropriate, including warning
+the offender or expulsion from the event (without refund, if applicable) and
+banning from future events.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+### Weapons Policy
+
+No weapons are allowed at ChiPy venues, including but not limited to explosives
+(including fireworks), guns, and large knives such as those used for hunting or
+display, as well as any other item used for the purpose of causing injury or
+harm to others. Anyone seen in possession of one of these items will be asked
+to leave immediately, and will only be allowed to return without the weapon.
+
+Attendees are further expected to comply with all state and local laws on this
+matter.
+
+## Scope
+
+All ChiPy attendees are subject to the Code of Conduct. This includes ChiPy
+organizers and volunteers, as well as attendees of the talk events,
+collaborative work environments, receptions / after-party events, or other
+sponsored or unofficial events related to ChiPy events.
+
+Sponsors are also subject to the anti-harassment policy. In particular,
+sponsors should not use sexualised images, activities, or other material.
+
+## Contact Information
+
+Anyone can report harassment. If someone’s behavior has made you uncomfortable,
+or if you witness the same happening to someone else, you should immediately
+contact ChiPy organizers in-person or on Slack, or send an email to the On-Call
+responser at chipy.conduct@gmail.com.
+
+Organizers of ChiPy will be happy to help participants contact hotel/venue
+security or local law enforcement, provide escorts, or otherwise assist any
+attendee to feel safe during ChiPy events. We value your attendance.
+
+## Procedure for Handling Incidents
+
+To respond to incidencts, those receiving requests will follow the [Organizers
+procedure for report handling](./report-handling-procedure)
+
+## Updating
+
+This Code of Conduct is a living document and subject to revision and change.
+Requests may be submitted for consideration. The final decision to incorporate
+changes or otherwise update this document is subject to a decision by the ChiPy
+Organizers.
+
+## License
+
+This Code of Conduct was forked from the example policy from the [PyCon US Code
+of
+Conduct](https://github.com/python/pycon-code-of-conduct/blob/master/code_of_conduct.md),
+which is under a Creative Commons Zero license.
```

Comment:
I think I saw a reference to Django's coc in some communication thread. unless I'm mistaken, that should be added here as well, right?

Reply:
The Code of Conduct doesn't use Django's, but the response guide does (and we do cite there).